.buildkite/commands/checkout-release-branch.sh

```diff
@@ -1,17 +1,11 @@
 #!/bin/bash -eu
 
-echo "--- :git: Checkout Release Branch"
-
-# Note: `BUILDKITE_RELEASE_VERSION` is the legacy environment variable passed to Buildkite by ReleaseV2.
-# It used the `BUILDKITE_` prefix so it was not filtered out when passed to the MacOS VMs, due to how `hostmgr` works.
-# This is considered legacy: we should eventually remove all use of custom `BUILDKITE_` variables, and instead
-# resolve the value of those sooner (i.e. in the YML pipeline) then pass it as parameter to the `.sh` calls instead.
+RELEASE_VERSION="${1:?RELEASE_VERSION parameter missing}"
 
-# Use the provided argument if there's one, otherwise fall back to the legacy BUILDKITE_RELEASE_VERSION
-RELEASE_VERSION=${1:-$BUILDKITE_RELEASE_VERSION}
+echo "--- :git: Checkout Release Branch"
 
 if [[ -z "${RELEASE_VERSION}" ]]; then
-    echo "RELEASE_VERSION is not set and BUILDKITE_RELEASE_VERSION is not available."
+    echo "RELEASE_VERSION is not set."
     exit 1
 fi
```

Comment:
```suggestion
```

This can be completely deleted, since if `${RELEASE_VERSION}` is empty, the script would have already failed and existed early on line 3.